tests/unit/synapseclient/core/multithread_download/unit_test_download_threads.py

```diff
@@ -131,20 +131,24 @@ def test_generate_chunk_ranges():
 
 
 def test_pre_signed_url_expiration_time():
+    # GIVEN a pre signed URL with an expiration date of: Jul 22 2013 20:12:07 in seconds since epoch UTC
     url = (
         "https://s3.amazonaws.com/examplebucket/test.txt"
         "?X-Amz-Algorithm=AWS4-HMAC-SHA256"
         "&X-Amz-Credential=your-access-key-id/20130721/us-east-1/s3/aws4_request"
-        "&X-Amz-Date=20130721T201207Z"
-        "&X-Amz-Expires=86400"
```

Comment:
Although on second thought... we support external buckets within Synapse and those buckets won't support this feature...

Reply:
I updated the logic for this